js/sia.js

```diff
@@ -21,12 +21,12 @@ function DaemonManager () {
     command: process.platform === 'win32' ? 'siad.exe' : 'siad',
     headers: {
       'User-Agent': 'Sia-Agent'
-    }
+    },
+    // Track if Daemon is running
+    running: false
   }
-  // Object to be returned
+  // Keep reference to `this`
```

Comment:
Just look at how it's used later? It's going to be confusing if you read line by line :/